backend/CertificateManagerAPIs/Controllers/SupplierController.cs

```diff
@@ -0,0 +1,46 @@
+﻿using CertificateManagerAPIs.DTO;
+using CertificateManagerAPIs.Services;
+using Microsoft.AspNetCore.Mvc;
+
+
+namespace CertificateManagerAPIs.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class SupplierController : Controller
+    {
+        private readonly ISupplierService _supplierService;
+
+        public SupplierController(ISupplierService supplierService)
+        {
+            _supplierService = supplierService;
+        }
+
+        [HttpGet("All suppliers")]
```

Comment:
method name must not contain empty spaces

Reply:
this is not solved

Reply:
Thank you, I thought that it was the space above it but now I got it.